boards/nxp/frdm_rw612/Kconfig.defconfig

```diff
@@ -5,6 +5,9 @@
 
 if BOARD_FRDM_RW612
 
+config FLASH_LOAD_SIZE
+	default 0x400000 if !BOOTLOADER_MCUBOOT && !(NXP_MONOLITHIC_WIFI || NXP_MONOLITHIC_BT || NXP_MONOLITHIC_IEEE802154)
```

Comment:
Guess, we have decided to avoid this FLASH_LOAD_SIZE change.

Reply:
>It should be used other way, e.g. code partition overlay, instead of FLASH_LOAD_SIZE change.
We have passed it during reviewing RWx WiFi PRs.
OK, I see.
> BTW: Does non-monolithic WiFi & BT work in Zephyr now?
Yes, set CONFIG_NXP_MONOLITHIC_WIFI=n, CONFIG_NXP_MONOLITHIC_BT=n, then it is non-monolithic build.